R/base64.R

```diff
@@ -117,10 +117,14 @@ process_html_res <- function(html, reg, processor) {
 }
 
 process_images <- function(html, processor) {
```

Comment:
`process_images()` is called by `html_document_base()`, so the change in this function has potential risk of affecting all HTML output formats. I'd rather be a little conservative, and move the new `process_html_res()` call you added to the end of `base64_encode_images()` (line `#140`), so that this only affects `ioslides_presentation()`, because only `ioslides_presentation()` calls `base64_encode_images()`.

Reply:
@yihui sorry I didn't see this update. No worries on the timing, I've been reporting using a branch that includes it in the intervening time. Good to see it merged. Thanks.

Reply:
No worries. Thanks!